src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -100,14 +115,18 @@ uct_rc_verbs_ep_rdma_zcopy(uct_rc_verbs_ep_t *ep, const uct_iov_t *iov,
     struct ibv_sge sge[UCT_IB_MAX_IOV];
     struct ibv_send_wr wr;
     size_t sge_cnt;
+    int send_flags = IBV_SEND_SIGNALED;
 
     UCT_RC_CHECK_RES(&iface->super, &ep->super);
     sge_cnt = uct_ib_verbs_sge_fill_iov(sge, iov, iovcnt);
     UCT_SKIP_ZERO_LENGTH(sge_cnt);
     UCT_RC_VERBS_FILL_RDMA_WR_IOV(wr, wr.opcode, opcode, sge, sge_cnt, remote_addr, rkey);
     wr.next = NULL;
+    if (opcode == IBV_WR_RDMA_READ) {
+        send_flags |= uct_rc_verbs_ep_atomic_fence(iface, ep, IBV_SEND_FENCE);
```

Comment:
what about get_bcopy?
maybe we can add it to common code, uct_rc_verbs_ep_post_send()?

Reply:
this comment is not addressed